tools/prepare-machine.ps1

```diff
@@ -239,8 +239,8 @@ if ($Cleanup) {
         # Verifier configuration: standard flags on all XDP components, and NDIS.
         # The NDIS verifier is required, otherwise allocations NDIS makes on
         # behalf of XDP components (e.g. NBLs) will not be verified.
-        Write-Verbose "verifier.exe /standard /driver xdp.sys fnmp.sys fnlwf.sys ndis.sys ebpfcore.sys"
-        verifier.exe /standard /driver xdp.sys fnmp.sys fnlwf.sys ndis.sys ebpfcore.sys | Write-Verbose
+        Write-Verbose "verifier.exe /standard /driver xdp.sys xdpfunctionaltestdrv.sys fnmp.sys fnlwf.sys ndis.sys ebpfcore.sys"
+        verifier.exe /standard /driver xdp.sys xdpfunctionaltestdrv.sys fnmp.sys fnlwf.sys ndis.sys ebpfcore.sys | Write-Verbose
```

Comment:
Just adding one driver makes prepare-machine.ps1 to fail.
Any idea? @nigriMSFT

Reply:
I wouldn't worry about adding the test driver atm. Just remove it to get unblocked.

Reply:
Please create a standalone PR that only adds `xdpfunctionaltestdrv.sys` to the list and updates https://github.com/microsoft/xdp-for-windows/blob/main/.azure/agents/functional.json. 

We should also update the documentation for our CI pools steps to deploy new agent JSONs. These are currently internal-only at [OneNote ](https://microsoft.sharepoint.com/teams/STACKTeam-CoreNetworkingMobileConnectivityPeripheralsStackSe/_layouts/OneNote.aspx?id=%2Fteams%2FSTACKTeam-CoreNetworkingMobileConnectivityPeripheralsStackSe%2FShared%20Documents%2FCore%20Networking%2FDPT-DataPathTransports%2FDPT%20Dev%20Notebook&wd=target%28Design%20Notes.one%7C5AF4BE33-1B57-4936-B3B1-A740A4335751%2FManaging%20CI%20Host%5C%2FAgents%7C844B71DF-8FE9-47C7-B9F3-59A1B4406E13%2F%29) but IMO should be merged into a `docs/internal` markdown file in GitHub.

Reply:
I created https://github.com/microsoft/xdp-for-windows/pull/668 to update the documentation and migrate to our repo.